docs/pages/hdmi_cec_source_tests.md

```diff
@@ -0,0 +1,179 @@
+
+- Boot with control configuration with various configurations having a predefined set of nodes:
+  - configuration to support the discovery of logical addresses. The caller to provide a create a proper logical address during Open and that should be provided when HDMICECGetLogicaladdress is used.
+  - Verify for the valid logical address and return the appropriate error code based on the logical address availability.  
+
+### Control Plane Requirements
+
+- The control plane will allow removing or adding a node to the network.
+  - allowing adding/removing source node
+  - Support the CEC commands from the external devices on L3 Test Cases. 
+
+## Physical Address
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+| 2| [Physical Address](#physical-address)| Enable the television connected to the DUT first to get the valid physical address allocated through the HAL function.|HdmiCecGetPhysicalAddress| Y  |NA||
+| | | Verify the physical addresses allocated by connecting a source and sink device through an HDMI switch.| HdmiCecGetPhysicalAddress| NA | Y  | Enable the television connected to `DUT` to declare its physical address first before `DUT`.|
+
+### Emulator Requirements - Physical Address
+
+- Boot control configuration to setup the CEC network nodes
+    - Emulator to provide a valid physical address when requested.
+
+### Control Plane Requirements - Physical Address
+
+- The control plane will allow removing or adding a node to the network.
+  - allowing add source node before the `DUT` switched ON.  
+  - allowing add sink node before the `DUT` switched ON.    
+
+## CEC Synchronous Transmission
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+| 3| [CEC Transmission](#cec-transmission)| Verify the correct transmission of the supported CEC commands (as per 1.4b HDMI CEC spec) to the connected device and ensure it is acknowledged properly.| HdmiCecTx | NA | Y  |Control plane to switch ON a CEC-supported device on the HDMI network so that it shall respond to the basic commands|
+| | | Broadcast a supported CEC Command to all the devices connected to the network without any error |HdmiCecTx| NA | Y  |Control plane to switch ON a CEC-supported device on the HDMI network to act on the broadcasted command|
+| | | Transmit a CEC Command (as per 1.4b HDMI CEC spec) to put the connected device into standby mode and await the device's response. Monitoring the behaviour of the connected device accordingly.| HdmiCecTx | NA | Y  | Control plane to monitor the behaviour of the connected devices.  |
+| | | Transmit a CEC Command (as per 1.4b HDMI CEC spec) to get the CEC Version for a logical address that doesn't exist.| HdmiCecTx | Y | NA  |  Control plane can unplug or switch off a previously existing CEC device |
```

Comment:
Should provide full details of what is API return and what is API status when called this function.

Reply:
Updated as requested.